tests/foreman/cli/test_contentview.py

```diff
@@ -4,9 +4,9 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: CLI
+:CaseComponent: ContentView
```

Comment:
Bugzilla component is called `ContentViews`, in plural. See testimony.yaml for reference.